### PR TITLE
changes for adding provision for making call to UDS via udsjs

### PR DESCRIPTION
**Pull request by pbathia**
modified:   Gruntfile.js
	modified:   app/ascension/controllers/caseQueue.js
	modified:   app/ascension/services/caseQueueService.js
	modified:   app/chromed.html
	new file:   app/common/services/udsService.js
	modified:   app/index.html
	modified:   app/loader.js
	modified:   bower.json
	modified:   karma.conf.js

### CODE REVIEW

**Comment by pbathia** [comment]
@sshumake Please review and merge.

File: app/common/services/udsService.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+'use strict';
```
Comment (sshumake) [inline app/common/services/udsService.js]: We should enable caching....however this is outside the scope of this pull request

File: bower.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
         "angular-ui-router": "0.2.8-bowratic-tedium",
         "angular-bootstrap": "0.10.0",
         "stratajs": "~1.1.14",
+        "udsjs":"*",
```
Comment (sshumake) [inline bower.json]: I'm ok with using * at the moment, but this needs to change once udsjs will have less churn.

**Comment by sshumake** [comment]
Looks good!

File: app/common/services/udsService.js (hunk 1 above)
Comment (pbathia) [inline app/common/services/udsService.js]: yes i agree, but i was of the opinion once we start pulling actual data from UDS, we will add caching, this will allow us better testing. So i kept the parameter as it is, as we wont forget it.
